src/tequila/__init__.py

```diff
@@ -5,7 +5,7 @@
     ExpectationValue, Variable, assign_variable, format_variable_dictionary,\
     vectorize
 
-from tequila.ml import to_platform,HAS_TORCH
+from tequila.ml import to_platform, HAS_TORCH, HAS_TF
```

Comment:
please delete
to_platform is ml specific and should best be called like
tq.ml.to_platform
so just import the to_platform in the __init__ of the ml directory

HAS_TORCH is not needed by users